### PR TITLE
updating db_storage and file_storage with methods: get and count

### PR DESCRIPTION
**Pull request by dairof7**

### CODE REVIEW

File: models/engine/db_storage.py [hunk 1]
```diff
@@ -74,3 +74,12 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """retrieve the object identified with class.id """
+        k = "{}.{}".format(cls.__name__, id)
```
Comment (bryano13) [inline models/engine/db_storage.py]: Please  specify what do you mean by naming that variable "k". Is it key?

File: models/engine/file_storage.py [hunk 2]
```diff
@@ -68,3 +68,12 @@ def delete(self, obj=None):
     def close(self):
         """call reload() method for deserializing the JSON file to objects"""
         self.reload()
+
+    def get(self, cls, id):
+        """retrieve the object identified with class.id """
+        k = "{}.{}".format(cls.__name__, id)
```
Comment (bryano13) [inline models/engine/file_storage.py]: what do you mean with the variable "k"

**Review by bryano13 — CHANGES_REQUESTED** [review]
Is fine but please correct one variable name.

File: models/engine/db_storage.py (hunk 1 above)
Comment (dairof7) [reply models/engine/db_storage.py]: sure, let me change it

File: models/engine/file_storage.py (hunk 2 above)
Comment (dairof7) [reply models/engine/file_storage.py]: sure, let me change it

**Comment by bryano13** [comment]
Two methods added (get and count) for filestorage and destorage. One correction made for  a variable name.

**Pull request merged**